src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -215,12 +220,18 @@ export class ServiceFormComponent extends CdForm implements OnInit {
   }
 
   ngOnInit(): void {
+    if (this.router.url.includes('services')) {
+      this.pageURL = 'services';
+    }
     this.action = this.actionLabels.CREATE;
+    this.hiddenServices = this.wizardStepService.hiddenServices;
```

Comment:
This is an improvement over the previous approach, but it's still too complicate...

Instead of making a lower level component (Service Page > Service Form) internally aware of a higher level abstraction (Cluster Expansion Wizard) now by means of this 3rd-party catch-all service (which breaks the encapsulation principle), why not using the regular top-to-bottom approach? That is, higher level component customize lower level components by sending `@Input` arguments?

https://medium.com/@izzatnadiri/how-to-pass-data-to-and-receive-from-ng-bootstrap-modals-916f2ad5d66e